photon-targeting/src/main/java/org/photonvision/targeting/PhotonTrackedTarget.java

```diff
@@ -127,12 +153,20 @@ public boolean equals(Object o) {
                 && Double.compare(that.area, area) == 0
                 && Objects.equals(bestCameraToTarget, that.bestCameraToTarget)
                 && Objects.equals(altCameraToTarget, that.altCameraToTarget)
-                && Objects.equals(targetCorners, that.targetCorners);
+                && Objects.equals(minAreaRectCorners, that.minAreaRectCorners)
+                && Objects.equals(detectedCorners, that.detectedCorners);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(yaw, pitch, area, bestCameraToTarget, altCameraToTarget);
+        return Objects.hash(
+                yaw,
+                pitch,
+                area,
+                bestCameraToTarget,
+                altCameraToTarget,
+                detectedCorners,
```

Comment:
Were we just not hashing `targetCorners` before??

Reply:
Not that I saw, no

Reply:
Yeah lmao